DOL.WHD.Section14c.Web/src/modules/components/mainNavigationControl/mainNavigationControlController.test.js

```diff
@@ -0,0 +1,31 @@
+describe('mainNavigationControlController', function() {
+
+    beforeEach(module('14c'));
+
+    beforeEach(inject(function ($rootScope, $controller, $route, navService) {
+        scope = $rootScope.$new();
+        route = $route;
+        mockNavService = navService;
+
+        spyOn(mockNavService,'hasNext');
+        spyOn(mockNavService,'hasBack');
+
+        route.current = {
+            params: { section_id : 1 }
+        }
+
+        spyOn(mockNavService,'getNextSection');
+
+        mainNavigationControlController = function() {
+            return $controller('mainNavigationControlController', {
+                '$scope': scope, 
+                'navService': mockNavService,
+                '$route': route
+            });
+        };
+    }));
+
+    it('invoke controller', function() {
+        var controller = mainNavigationControlController();
```

Comment:
updated directive unit tests to assert it was created https://github.com/AppliedIS/dol-whd-14c/commit/e73d8d8cc8291dbbcf47a7da435a7ad963cc54a4